src/mixins/Observable.ts

```diff
@@ -0,0 +1,142 @@
+//@ts-nocheck
+
+type EventRegistryObject = { [eventName: string]: Function };
```

Comment:
I think there is a a TS util, Record, that makes this more clear.

Reply:
great